compiler/luci-interpreter/src/kernels/BroadcastTo.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kernels/BroadcastTo.h"
+#include "kernels/Utils.h"
+
+#include "PALBroadcastTo.h"
+
+#include <stdexcept>
+#include <iostream>
+
+namespace luci_interpreter
+{
+
+namespace kernels
+{
+
+static Shape extractShapeFromTensor(const Tensor *tensor)
+{
+  Shape shape(tensor->shape().num_elements());
+  if (tensor->element_type() == DataType::S32)
+  {
+    const auto *shape_data = tensor->data<int32_t>();
+    for (int i = 0; i < tensor->shape().num_elements(); ++i)
+    {
+      shape.dim(i) = shape_data[i];
```

Comment:
Q) is there no cases where shape_data[i] < 0 ?

Reply:
Plz check tflite kernel and how it does.

Reply:
@seanshpark 

with the case of `shape_data[i] < 0`, I set the args value in the recipe file and proceeded with the build and test, but it worked well without failure. 
Should I add a new case where shape_data[i] < 0 ?

Reply:
Have to checked tflite kernel source codes with BroadcastTo shape input?

Reply:
> Have to checked tflite kernel source codes with BroadcastTo shape input?

Perhaps if the dim of `broadcast_to` shape input becomes negative number, the dim value will be automatically adjusted by other dim values of the shape.

I found this in the numpy library.
For the first time, I declared a numpy array with a shape (1, 4, 3), and then I recreated it with (-4, 6) using `Reshape`. 

Therefore, it was confirmed that the dim value of the negative number was automatically dynamically calculated by the later 6 dim value and converted to (2, 6).

The following is the code, and the value output for this.

- python code
```
import numpy as np

A = np.array([[[1, 2, 3], [1, 2, 3], [1, 2, 3], [1, 2, 3]]])

print(np.shape(A))

A.shape = (-4, 6)
print(A)

print(np.shape(A))
```

- print log
```
(1, 4, 3)
[[1 2 3 1 2 3]
 [1 2 3 1 2 3]]
(2, 6)
```

In addition, the `shape` of `broadcast_to` in the `tensorflow` follows the characteristics of the `tf.shape` of the tensorflow, and when the dim of the `tf.shape` is negative number or `None`, the value of dim is dynamically changed like the characteristics of the `numpy`.
-> [reference](https://www.tensorflow.org/api_docs/python/tf/shape)

So I think i should add assign codes a negative value in this function to prevent it from entering the shape dim value.

So I think i need to add an `assert` code to prevent negative values from being set in the shape dim value in this function. :)